README.rst

```diff
@@ -0,0 +1,53 @@
+=============
+django-stripe
+=============
+|travis|_ |pypi|_ |coveralls|_ |requiresio|_
+
+Stripe integration for Django-based projects
+
+This project's target is to make the Stripe API easier to use in Django-based applications.
+At the moment the library supports:
+
+* charging users
+
+Installation
+============
+Add ``django_stripe`` to your app's ``INSTALLED_APPS``, and also set ``STRIPE_API_KEY`` in project settings. After all please migrate the app (``./manage.py migrate django_stripe``).
+
+Usage
+=====
+
+Charging
+--------
+First of all, make sure to obtain Stripe user token from the Stripe API, and then save it to ``django_stripe.models.StripeToken``, for example:
+::
+  
+  customer = stripe.Customer.create(source=data["id"])
+  token = StripeToken.objects.create(user=request.user, content=data, # JSON response from Stripe API (in front-end) 
+                                    customer_id=customer["id"])
```

Comment:
@remik the code is self explanatory, you have two commands, one gives customer. 
@poxip, please keep improving it as TODO. RIght now it will block other integrations.